src/status_im2/common/standard_auth/view.cljs

```diff
@@ -0,0 +1,154 @@
+(ns status-im2.common.standard-auth.view
+  (:require [react-native.core :as rn]
+            [quo2.components.numbered-keyboard.numbered-keyboard.view :as numbered-keyboard]
+            [quo2.core :as quo]
+            [status-im2.common.standard-auth.style :as style]
+            [reagent.core :as reagent]
+            [quo2.foundations.colors :as colors]
+            [react-native.fast-image :as fast-image]
+            [status-im2.common.resources :as resources]
+            [utils.i18n :as i18n]
+            [quo2.components.keycard.view :as keycard]
+            [utils.re-frame :as rf]))
+
+(defn enter-keycard-pin-sheet
+  []
+  (let [entered-numbers (reagent/atom [])
+        max-digits 6
+        max-attempt-reached (reagent/atom false)]
+    (fn []
+      (defn handle-number-entry [number]
+        (when (>= (count @entered-numbers) max-digits)
+          (reset! entered-numbers []))
+        (when (= (count @entered-numbers) (dec max-digits))
+          (reset! max-attempt-reached true))
+        (swap! entered-numbers conj number))
+
+      (println "entered-numbers" @entered-numbers)
+
+      (defn get-dot-style [idx]
```

Comment:
could you please elaborate on why defn inside function ?

Reply:
@flexsurfer I'm new to ClojureScript's syntax so that was why. Now I got it and fixed it by defining functions using `fn` within the `let` block.